packages/portal/tests/portal.test.tsx

```diff
@@ -9,7 +9,7 @@ test("should render portal", () => {
     </PortalManager>,
   )
 
-  expect(tools.baseElement).toMatchSnapshot()
+  expect(tools.baseElement.innerHTML).toMatchSnapshot()
```

Comment:
idk whats up with that test. locally it doesnt work, then I fix it, push, and CI wants the initial version. then I reverse, and CI still complains.